ui/packages/catalog-ui-search/src/main/webapp/react-component/map-info/container.tsx

```diff
@@ -50,15 +50,12 @@ const getAttributes = (map: Backbone.Model) => {
   }
   return properties.summaryShow
     .map((attribute: string) => {
-      const definition = metacardDefinitions.metacardTypes[attribute]
-      const name =
-        definition !== undefined ? definition.alias || definition.id : attribute
       const value = map
         .get('targetMetacard')
         .get('metacard')
         .get('properties')
-        .get(name)
-      return { name, value }
+        .get(attribute)
+      return { name: attribute, value }
```

Comment:
We should still grab the value via the `attribute` name, but we need to fetch the definition for the metacard, and if it exists, grab the alias and return it here instead, otherwise we can get bad attribute display names here.

Reply:
I did that originally but you can see in `formatAttribute` below that it also grabs the metacard type from definitions. In practice, this causes an error if I return the alias here because in format attribute it will then try and find the definition via alias, and it won't find it. I am leaving alias retrieval until the last point it's needed, at render (which is done in `formatAttribute`)